includes/Plugin.php

```diff
@@ -180,6 +180,7 @@ function( $username, $user ) use ( $user_options ) {
 		( new Core\Util\Activation( $this->context, $options, $assets ) )->register();
 		( new Core\Util\Migration_1_3_0( $this->context, $options ) )->register();
 		( new Core\Util\Reset( $this->context ) )->register();
+		( new Core\Util\Debug_Data( $this->context ) )->register();
```

Comment:
This could be moved into the `init` closure to then receive the other dependencies (see above comment).